utilities/ovn-ctl

```diff
@@ -212,14 +214,23 @@ $sync_from_port" > $active_conf_file
     fi
 
     if test $mode = cluster; then
+        # check for election timer arg support in ovsdb-tool
+        has_election_timer=$(ovsdb-tool --help | grep "\--election-timer")
```

Comment:
NIT: add --quiet
`has_election_timer=$(ovsdb-tool --help | grep -q "\--election-timer")`

Reply:
@flavio-fernandes I was acutally depending on the output for the `if test -n $has_election_timer` bit just below... what's the right way to use the exit code 0/1 when testing?

Reply:
@flavio-fernandes I think I'd like to leave it as-is; I tried using the command in the `if` and I think the result is less clear:

```
        # check for election timer arg support in ovsdb-tool
        if test -n "$ovn_db_election_timer" &&
            ! ovsdb-tool --help | grep "\--election-timer"; then
            # caller requested election timer but ovsdb-tool doesn't support it
```

Reply:
> @flavio-fernandes I was acutally depending on the output for the `if test -n $has_election_timer` bit just below... what's the right way to use the exit code 0/1 when testing?

ah, I see. sorry for missing that.
`if test -n "$ovn_db_election_timer"` is a good approach then.